components/MeasureSelect.tsx

```diff
@@ -0,0 +1,37 @@
+import { Select, SelectItem } from '@mantine/core';
+import { showNotification } from '@mantine/notifications';
+import { Component, useEffect, useState } from 'react';
+import { AlertCircle } from 'tabler-icons-react';
+
+export default function MeasureSelect() {
+  const [measures, setMeasures] = useState<SelectItem[]>([]);
+  const [selectedMeasure, setSelectedMeasure] = useState<string | null>(null);
+
+  useEffect(() => {
+    fetch(`${process.env.NEXT_PUBLIC_DEQM_SERVER}/Measure`)
```

Comment:
Can we add a .env.local file to define a default value for this?

Reply:
I believe next ignores `.env*.local` by default, I'm assuming this is best practice.

Rather, could we commit a `.env.example` file or something with an example.com URL as the `NEXT_PUBLIC_DEQM_SERVER`? And then in the READE we can say to rename the file to .env.local and replace the value with where deqm-test-server is running?

This also avoids us potentially commiting an internal hostname to a public repo

Reply:
Oh man. I just noticed that it did `.gitignore` the `.env.local` file. Sorry about this confusion. Will work on adding the example `.env` and README info.

Reply:
Added example and info in README in 1ef199d.